tests/cephfs/cephfs_cbt.py

```diff
@@ -0,0 +1,42 @@
+import random
+import string
+import traceback
+
+from cli.io.cbt.cbt import CBT
+from tests.cephfs.cephfs_utilsV1 import FsUtils
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    try:
+        log.info(f"MetaData Information {log.metadata} in {__name__}")
+        fs_util = FsUtils(ceph_cluster)
+        config = kw.get("config")
+        build = config.get("build", config.get("rhbuild"))
+        clients = ceph_cluster.get_ceph_objects("client")
+        installer = ceph_cluster.get_ceph_objects("installer")
+        fs_util.prepare_clients(clients, build)
+        fs_util.auth_list(clients)
+        mon = ceph_cluster.get_ceph_objects("mon")
+        osds = ceph_cluster.get_nodes("osd")
+        osd_list = [osd.hostname for osd in osds]
+        benchmarks = config.get("cbt_benchmarks")
+        a = CBT(installer[0])
+        a.setup_CBT()
```

Comment:
wdyt about calling `setup_CBT` from `__init__` method as primary_client is known.